feature-utils/poly-look/jsdoc.conf.json

```diff
@@ -0,0 +1,25 @@
+{
+    "tags": {
+      "allowUnknownTags": true,
+      "dictionaries": ["jsdoc", "closure"]
+    },
+    "source": {
+      "include": ["./src"],
+      "includePattern": "\\.(jsx|js|ts|tsx)$",
+      "excludePattern": "(^|\\/|\\\\)_"
+    },
+    "plugins": [ "plugins/markdown", "../../node_modules/better-docs/" ],
+    "templates": {
+      "better-docs": {
+        "name": "poly-look documentation"
+      }
+    },
+    "opts": {
+      "template": "../../node_modules/better-docs",
```

Comment:
our own tool to handle react components? what do you mean?

Reply:
A template that's in the repo, and not generated by the install process.

Reply:
well this would mean a rewriting of `better-docs` tool... if I get you correctly. haven't seen anywhere doing it that way tbh

Reply:
OK, I don't think we're on the same page here. So let's start over. What exactly does this configuration do?

Reply:
we need the `better-docs` in order to support the `.jsx` documentation... it's not properly handle by `jsdoc` with what we've till now..

Reply:
OK, I get now that what handles the .jsx files is this other module. But I circle back to my initial objection: is that now installed in some path-accessible place? Isn't it run via npx so that you can access it in its binary directory? Can you change that to `npx better-docs` so that you don't depend on specific paths?

Reply:
It didn't work first hand - will look at it further at a next PR as it's fine for you this one